geoapi/tasks/external_data.py

```diff
@@ -144,6 +144,8 @@ def import_point_clouds_from_agave(userId: int, files, pointCloudId: int):
         point_cloud = pointcloud.PointCloudService.get(session, pointCloudId)
         celery_task_id = celery_uuid()
 
+        logger.info(f"point cloud:{pointCloudId} conversion started for user:{user.username} and files:{files}")
```

Comment:
this is the main change.  it is just another logging statement as on the worker-side of things there was no log statement when the point cloud processing began which made looking at logs a bit tricky.